content/posts/newsletter/Newsletter for 2020-01-10.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Happy new XMPP year! 10 Jan 2020
+save_as: 2020/01/newsletter-10-january/index.html
+url: 2020/01/newsletter-10-january/
+date: 2020-01-10 15:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+Happy new XMPP year!
+
+Welcome to the XMPP newsletter covering the month of December 2019.
+
+Contribute to the community effort, which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## Articles 
+
+[Whatsapp, Matrix or XMPP](https://www.golem.de/news/whatsapp-matrix-oder-xmpp-bmi-sucht-einen-messenger-fuer-bundesbehoerden-1912-145326.html) (german) - The German Federal Ministry of the Interior is looking for a messenger for the federal authorities while the German Federal Police is experimenting with XMPP. They are using an [Open Source Messenger "MOKA" based on XMPP](https://dipbt.bundestag.de/dip21/btd/19/152/1915218.pdf) (german).
```

Comment:
`based on Conversations` ?

Ref: https://nitter.net/iNPUTmice/status/1203611711967813633

Reply:
Also, you don't include the OMEMO posts ( https://monal.im/blog/omemo-fixes-catalyst-and-france/ & https://monal.im/blog/omemo-is-broken-in-general-across-the-ecosystem/ ) because they're not in December? 
I find them pretty important though since they have implications in the whole ecosystem and this needs faster coordination between multiple projects, hence the news needs to be put out there.